crates/uv/src/commands/project/init.rs

```diff
@@ -843,7 +870,9 @@ impl InitProjectKind {
         fs_err::write(path.join("pyproject.toml"), pyproject)?;
 
         // Generate `src` files
-        generate_package_scripts(name, path, build_backend, true)?;
+        if !bare {
+            generate_package_scripts(name, path, build_backend, true)?;
+        };
 
         // Initialize the version control system.
         init_vcs(path, vcs)?;
```

Comment:
Wait hold on shouldn't these be wrapped too (in a few places?)

Reply:
It happens up at https://github.com/astral-sh/uv/pull/11192/files#diff-a459ffa6a304d440657379d39bdef792a801589e6f6cbf1ae01b50754ab5fa98R262

A little questionable, but makes the `--vcs git` behavior easier.